README.md

```diff
@@ -69,6 +69,22 @@ Install Kerberos client to enable `skt` authentication to Beaker:
 Install `skt` dependencies following its
 [README.md](https://github.com/RH-FMK/skt).
 
+Run tests
+---------
+
+For running all tests write down:
+
+    python -m unittest discover tests
```

Comment:
We are using `pytest` for sktm, so both this and the line below should be modified to reflect on that.